src/components/products/review/CreateReview.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import submitProductReview from '../../../redux/actions/ratingAndFeebackActions';
+import StarRating from './StarRatings';
+
+const CreateReview = ({ productId }) => {
+  const [rating, setRating] = useState();
+  const [feedback, setFeedback] = useState('');
+  const [isSubmitted, setIsSubmitted] = useState(false);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    const storedStatus = localStorage.getItem('reviewSubmitted');
+    if (storedStatus === 'true') {
+      setIsSubmitted(true);
+    }
+    return () => {
+      localStorage.removeItem('reviewSubmitted');
+    };
```

Comment:
I think we decide to remove this one (localStorage related) coz we found that if a user changes the device or browser this can not work anymore.

Reply:
yes, fixed